package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ainblockchain/ain-util",
-  "version": "1.1.7",
+  "version": "1.1.8",
```

Comment:
it seems that we need to upgrade the version too frequently because we're merging to master by default.. should we set up git-flow for this repo & ain-js? (not the scope of this PR but for the future~)

Reply:
The last one is `patch` which is able to be upgraded fast enough :) No worries! github-flow seems to be enough for ain-util. Let's talk about it offline !!